src/js/common/schemaform/FormPage.jsx

```diff
@@ -94,13 +100,32 @@ class FormPage extends React.Component {
     this.props.router.push(pages[page].path);
   }
 
+  handleSave() {
+    const {
+      formId,
+      version,
+      data
+    } = this.props.form;
+    const returnUrl = this.props.location.pathname;
+    this.props.saveInProgressForm(formId, version, returnUrl, data);
+
+    // TODO: Build this page and make it accessible (and customizable) to all forms
+    // this.router.push('/form-saved');
+  }
+
   render() {
-    const { route, params, form } = this.props;
+    const { route, params } = this.props;
```

Comment:
Mmm...I think it was only used once before I merged master back in, but I don't remember.